(qwikcity)/qwikcity/index.mdx

```diff
@@ -11,37 +11,37 @@ contributors:
   - adamdbradley
 ---
 
-# Qwik City (routing)
+# Qwik City (enrutamiento)
 
-_aka. Qwik Router_
+_también conocido como Qwik Router_
 
-Learning Qwik and Qwik City are not two different things, Qwik City is built on top of Qwik, bringing opinionated routing and goodies to make your life easier.
+Aprender Qwik y Qwik City no son dos cosas diferentes, Qwik City se construye sobre Qwik, brindando un enrutamiento y características opinionadas para facilitar tu vida.
```

Comment:
Según lo que entiendo al final de esta línea "bringing opinionated routing and goodies to make your life easier", se refiere a que implementa un enrutamiento disruptivo, un cambio brusco y nuevo, algo fresco y obstinado.

Puedo recomendar quizá algo como:

Qwik City está construido sobre Qwik, trayendo un enrutamiento disruptivo y cositas que facilitarán tu vida.

Reply:
Qwik City está construido sobre Qwik, incorporando un enrutamiento y características con prácticas bien definidas que facilitarán tu vida.

Esto es lo que voy a poner, si te parece bien